scss/components/table.scss

```diff
@@ -89,7 +93,8 @@ $block: #{$fd-namespace}-table;
         }
         border-style: solid;
         border-width: $fd-table-border-width;
-        border-color: $fd-table-border-color;
+        border-color: red;
```

Comment:
is this red color intentional?

Reply:
@saadmhybris sorry this was an error, will update it